src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/NetworkListener.java

```diff
@@ -39,4 +45,12 @@ public void onBlockBreak(BlockBreakEvent e) {
     public void onBlockPlace(BlockPlaceEvent e) {
         manager.updateAllNetworks(e.getBlock().getLocation());
     }
+
+    @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
+    public void onExplosiveToolUse(ExplosiveToolBreakBlocksEvent e) {
+        // Fixes #3013 - Also update networks when using an explosive tool
+        for (Block b : e.getAdditionalBlocks()) {
+            manager.updateAllNetworks(b.getLocation());
```

Comment:
is it quicker to run this for all blocks rather than checking if it's an SF block first... I can't see checking if it's an SF block being more intensive honestly, that's a Map lookup whereas this loops a list and does all sorts of crap